spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesServiceInstance.java

```diff
@@ -37,43 +34,49 @@
 
 	private static final String DSL = "//";
 
-	private static final String COLN = ":";
+	private static final String COLON = ":";
+
+	private String instanceId;
 
-	private final String instanceId;
```

Comment:
Let's keep these fields final as they were. I see a no-args constructor has been added, but it's only used in tests. Let's rather delete that constructor and create a test util method to easily create empty instances of this object.

Reply:
Fixed